hearts/hearts.py

```diff
@@ -14,11 +14,40 @@ def __hash__(self):
 
 class Game(object):
     def __init___(self, players, points_to_win=100):
-        self.points_to_win = points_to_win
+        self.max_points = points_to_win
         self.players = players
-        self.round_counter = 1
+        self.rounds = []
+        self.round_number = 1
+
         shuffle(self.players)
 
+    def create_round(self):
+        pass_direction = {0: 'keep', 1: 'left', 2: 'right', 3: 'across'}
+        new_round = Round(self.players, pass_direction[self.round_number % 4])
+        self.rounds.append(new_round)
+
+    @property
+    def scores(self):
+        # Returns a dict of the form {round_number: {Player: score}}
+        scores = {1: {player: 0 for player in self.players}}
+        for index, the_round in self.rounds.enumerate():
+            round_number = index + 1
+            if the_round.is_over:
+                scores[round_number] = the_round.final_scores()
+        return scores
+
+    @property
+    def total_scores(self):
+        # Game --> {Player: int}
+        totals = {player: 0 for player in self.players}
+        for round_score in self.scores.values():
+            for player, player_score in round_score.items():
+                totals[player] += player_score
+        return totals
+
+    def is_over(self):
+        return any(score >= self.max_points for score in self.total_scores.values())
```

Comment:
This is good. Another option (maybe shorter): `max(self.total_scores.values()) >= self.max_points`